src/windows/top_toolbar.c

```diff
@@ -1461,10 +1462,11 @@ static void window_top_toolbar_scenery_tool_down(short x, short y, rct_window *w
 	case SCENERY_TYPE_SMALL:
 	{
 		int quantity = 1;
-		if (window_scenery_is_build_cluster_tool_on) {
-			quantity = 35;
+		bool isCluster = window_scenery_is_build_cluster_tool_on && network_can_perform_action(network_get_current_player_group_index(), GAME_COMMAND_TOGGLE_SCENERY_CLUSTER);
+		if (isCluster) {
+			quantity = 36;
```

Comment:
@IntelOrca  forreal, bro? I'll restart into windows 8 now and try getting it in there then. Outta curiosity, are you going to put that disable litter in as well?

I also have a question, as I'm new with c a bit and i'm trying to understand the concept of the networking, could you explain how it works? What are the connection steps done with networkconnection, networkaddress, networkplayer, etc?